server/sockets.ts

```diff
@@ -495,6 +504,11 @@ export class ServerStream extends Streams.ObjectReadWriteStream<string> {
 				}
 			}
 		}
+		if (Config.validateSocket && Config.validateSocket.call(this, socket, socketip) === false) {
+			socket.destroy();
+			this.sockets.delete(socketid);
+			return;
+		}
```

Comment:
On further consideration, I still want this removed. This needs to be implemented in the main process, not here.

```suggestion
```

Reply:
(The logger can remain; it has plenty of other uses for debugging.)